docs/source/intro/tutorial.rst

```diff
@@ -570,6 +570,17 @@ Depending on which step the workflow is running, you should get something like t
 We can see that the ``MultiplyAddWorkChain`` is currently waiting for its *child process*, the ``ArithmeticAddCalculation``, to finish.
 Check the process list again for *all* processes (You should know how by now!).
 After about half a minute, all the processes should be in the ``Finished`` state.
+You can also see a nice overview of the processes called by the work chain with the ``verdi process status`` command:
```

Comment:
```suggestion
The ``verdi process status`` command prints a *hierarchical* overview of the processes called by the work chain:
```

I would usually leave it up to the user to decide whether the view is nice, but that's just a matter of taste.